### PR TITLE
doc: fix some typos and update outdated description

### PR DESCRIPTION
**Pull request by yaojingguo**
fix some typos and update outdated description.

Release note: None

### CODE REVIEW

File: docs/tech-notes/sql.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ For example, if the architecture calls out a thing called e.g. "query
 runner", which takes as input a logical query plan (a data structure)
 and outputs result rows (another data structure), you'd usually expect
 a thing in the source code called "query runner" that looks like a
-class whose instances would carry the execution's internal state, a
+class whose instances would carry the execution's internal state,
 providing some methods that take a logical plan as input, and returning
```
Comment (dchenk) [inline docs/tech-notes/sql.md]: I'd take the comma out after "input"—it's unnecessary grammatically.

Comment (yaojingguo) [reply docs/tech-notes/sql.md]: Yes, you are right.  Fixed in a new commit.

**Comment by justinj** [comment]
Looks good, thanks!

bors r+

**Comment by justinj** [comment]
Hmm, let's try that again

bors r+

**Pull request merged**